Kitodo/src/main/webapp/WEB-INF/resources/js/modeler-custom.js

```diff
@@ -9,7 +9,7 @@
  * GPL3-License.txt file that was distributed with this source code.
  */
 
- var availableUserRoles = [];
```

Comment:
Change in format. I think this file isn't even touched!?

Reply:
I removed an empty space in front of the declaration. Now it's correctly formatted.